### ISSUE TITLE
Add password rules for massciportal.com

### ISSUE THREAD

**Issue by gpscar** [apple/password-manager-resources#570]
https://patient.massciportal.com/

Must be at least 12 characters long.
Contain at least one upper case character.
Contain at least one lower case character.
Contain at least one number.
Contain at least one of the following characters: !, @, #, $, %, ^, &.
Must be different from your last 8 passwords.

**Comment by mangeshraut712** [comment]
`patient.massciportal.com` already has password rules on `main`:

`minlength: 12; required: upper; required: lower; required: digit; required: [!@#$%^&];`

This issue can likely be closed.

**Comment by gpscar** [comment]
Yeah the fix was made a month ago. @rmondello I should have claimed radar credit while I could.

**Comment by rmondello** [comment]
:P
